pkg/lib/config/authenticator_test.go

```diff
@@ -9,212 +9,37 @@ import (
 	"github.com/authgear/authgear-server/pkg/lib/config"
 )
 
-func TestValidPeriodDeprecateFlow(t *testing.T) {
-	Convey("Email Code Mode", t, func() {
-		test := func(inputYAML string, mode string) {
-			cfg, err := config.Parse([]byte(inputYAML))
-			So(err, ShouldBeNil)
-
-      var data []byte
-      switch mode {
-      case "code":
-        data, err = os.ReadFile("testdata/code_validperiod_test.yaml")
-        if err != nil {
-          panic(err)
-        }
-      case "login_link":
-        data, err = os.ReadFile("testdata/link_validperiod_test.yaml")
+func TestAuthenticatorCodeValidPeriodDeprecation(t *testing.T) {
+  Convey("authenticator_code_valid_period_deprecation", t, func() {
+    entries, err := os.ReadDir("testdata/authenticator_code_valid_period_deprecation/input")
```

Comment:
Use `path/filepath.Glob("testdata/authenticator_code_valid_period_deprecation/*.input.yaml")` instead, if you find the output os.ReadDir requires you further process them

Reply:
Updated thanks